puppeteer-scraper/package.json

```diff
@@ -4,11 +4,12 @@
   "description": "Crawl web pages using Apify, headless Chrome and Puppeteer",
   "main": "src/main.js",
   "dependencies": {
-    "@apify/scraper-tools": "^0.1.13",
-    "apify": "0.21.9"
+    "@apify/scraper-tools": "^0.1.14-dev.0",
+    "apify": "1.2.0-beta.5",
+    "puppeteer": "^8.0.0"
   },
   "devDependencies": {
-    "@types/puppeteer": "^3.0.0",
+    "@types/puppeteer": "^5.4.3",
```

Comment:
Do we need this with `puppeteer@8`?

Reply:
Actually puppeteer is not used anywhere in the code, it feels weird to have it in the deps in the first place, but it is required as peer dependency for the SDK in this context, right? Removed the typings as that is not needed for sure.

Reply:
https://github.com/apify/apify-js/issues/1005